UniversalScannerService/Program.cs

```diff
@@ -0,0 +1,39 @@
+using System;

```

Comment:
Could you please kindly explain what is the added value to use Grpc on a small tool like this? I am afraid I do not see a real added value here.